### DISCUSSION TITLE
<meta name="thumbnail" /> support

### DISCUSSION THREAD

**Discussion by martpie** [vercel/next.js#73402] [Ideas]
### Goals

1. Allow users to set thumbnails for their search results.

https://support.google.com/programmable-search/answer/1626955?hl=en

![Screenshot 2024-12-01 at 21 34 36](https://github.com/user-attachments/assets/32fd0259-fd81-4bfd-bb10-3708f862111b)

### Background

This would allow for website to showcase product picture in search results, or for an artist to showcase a piece of art.

I don't know if `og:image` is an equivalent, but it does not seem documented by Google, so I assume they are different (thumbnail seem to be squared)

### Proposal

Support a `thumbnail` value in the `Metadata` object, this would render a `<meta name="thumbnail" content="..." />` in the `<head>` section of the page.

**Comment by Netail** [comment]
Never heard of the `thumbnail` meta name, but `og:image` is what you are looking for, also the one Wikipedia is using. I cannot find a reference to a `thumnail` meta tag in Wikipedia's head

**Comment by puku0x** [comment]
```ts
export const metadata: Metadata = {
  other: {
    thumbnail: 'https://nextjs.org/og.png'
  },
};
```
